src/app/screens/connectors/ConnectUmbrel/index.tsx

```diff
@@ -31,7 +31,8 @@ export default function ConnectUmbrel() {
       let lndconnect = new URL(lndconnectUrl);
       lndconnect.protocol = "http:";
       lndconnect = new URL(lndconnect.toString());
-      const url = `https://${lndconnect.host}${lndconnect.pathname}`;
+      const pathname = lndconnect.pathname.replace(/\/$/, "");
```

Comment:
is this `$` correct - that's the end of the line? 

but where does the double slash actually come from? `.host` should not have a slash but `.pathname` starts with a slash. 

so I am wondeing where the double slash comes from. 🤔

Reply:
@bumi See my review comment.